packages/generator-react-server/test/app.js

```diff
@@ -24,6 +24,46 @@ test('generator-react-server:app creates default files', async t => {
 	t.false(await exists('docker-compose.yml', testDir));
 });
 
+test('generator-react-server:app .reactserverrc matches react-server-cli default option keys', async t => {
+	let testDir;
+	await helpers.run(path.join(__dirname, '../generators/app'))
+		.inTmpDir(dir => {
+			testDir = dir;
+		})
+		.withPrompts({name: 'foo', dockerCfg: false})
+		.toPromise();
+
+	// Reading files here instead of requiring because we don't understand export default here.
+	const defaultOptions = await readFile('defaultOptions.js', path.join(__dirname, '../../react-server-cli/src'));
```

Comment:
I completely agree but when I try and require it I get this error:

```
# generator-react-server:app .reactserverrc matches react-server-cli default option keys
not ok 1 - failed with "Unexpected token export"
  ---
    operator: undefined
    expected: undefined
    actual: undefined
    at: SyntaxError: Unexpected token export
  ...
```

Thoughts? There's gotta be a babel plugin different between react-server-cli and the generator but I couldn't quite figure it out and didn't really want to add a package for a unit test.